### PR TITLE
 Test models

### PR DESCRIPTION
**Pull request by hoangnh1211**
## Related Tickets
- https://edu-redmine.sun-asterisk.vn/issues/41265

## WHAT
-  Test models

## HOW
N/A

## WHY
-N/A

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by hoangnh1211** [comment]
ready

File: shop/tests/test_models.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+from django.test import TestCase, Client
+from django.urls import reverse
+from shop.models import CustomUser, Image, Item, Order, Product,Category, ProductSize, Size, Comment, Sale, SaleProduct
+from ..utils.constant import LENGTH_PAGE, STATUS_ORDER, FILTER, SALE
+
+class TestModel(TestCase):
```
Comment (huongnt-2545) [inline shop/tests/test_models.py:6]: Mỗi model thì em viết 1 class test riêng nhé

**Comment by hoangnh1211** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**